kubemarine/core/flow.py

```diff
@@ -288,7 +288,7 @@ def new_common_parser(cli_help):
                         help='auto-generated ansible-compatible inventory file location')
 
     parser.add_argument('--dump-location',
-                        default='./dump/',
+                        default='./',
```

Comment:
It is necessary to adopt existing usages of --dump-location, for example in `.github/workflows/test_install.yaml`. And in general, it is an incompatible change of API. The users should be notified in release notes to adopt their probable usages of the option.

Reply:
`.github/workflows/test_install.yaml` has been updated